spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
I still don't think a comment is the right choice here - if it's to be included, it should be an actual informative note. But I think the should be in HTML, not in ECMA262. Essentially any change to ECMA262 that adds observable behavior has a chance of violating the constraints (security or otherwise) of an ECMAScript embedding. I don't think this one special case is worth documenting in ECMA262 unless there is something particularly onerous here.